docs/howto/custom-shell.txt

```diff
@@ -39,8 +39,9 @@ For example:
 
 The customization above adds :func:`~django.urls.resolve` and
 :func:`~django.urls.reverse` to the default namespace, which already includes
-all models from the apps listed in :setting:`INSTALLED_APPS`. These objects
-will be available in the ``shell`` without requiring a manual import.
+all models from the apps listed in :setting:`INSTALLED_APPS` plus what is 
+imported by default. These objects will be available in the ``shell`` without 
+requiring a manual import.
 
 Running this customized ``shell`` command with ``verbosity=2`` would show:
```

Comment:
The console output needs to be updated below.
Under that, I think add the `.. versionchanged:: 6.0` which lists each utility that was added